### DISCUSSION TITLE
Feature Request : Enity displays aggreate_func value

### DISCUSSION THREAD

**Discussion by ozMerv** [kalkih/mini-graph-card#1158] [Ideas]
Is there a way to have the agreate_func value displayed on the card rather that the initial sensor value.

eg. When useing an energy meter that reads data as a total use. By using the aggrate_func and group_by you can read the the total daily energry comsupmtion and disply that as a graph. The figure is avaiable to you on mouse over. The figure on the Card under the name is the Sensor data - which is the total ( this could be months of data ).

**Comment by ildar170975** [comment]
aggreate_func Is const defined by a user. Means - you can display the value in the title, for example.

**Comment by ozMerv** [comment]
Thanks for the responce..   Can you explain or give me an example how I can display the value in the title.. ?

Attached is what I am using ( 2 pix ) .. one shows the total_energy in the title.. the second pix, shows the value when you mouse over the graph.  You can see from this that the aggregate_func is clearly working and producing the graph perfectly. I just cannot work out howe to make the cuurent aggregate_func value display in the title.

( I have tried using two different entities with different aggregate_func, but it still only shows the sensor data before the aggregate_func is calculated )
![aggerate-1](https://github.com/user-attachments/assets/fad6f5a3-9c53-43d6-8896-e49c14a1dbc7)
![aggerate-2](https://github.com/user-attachments/assets/2f7ebbcf-8871-45e6-bcb6-d3a614a340b2)

**Reply by ildar170975** [reply]
> can display the value in the title.. ?

If you just need to show a value of `aggregate_func` when it is `last`:
```
type: custom:mini-graph-card
title: "aggregate_func: last"
...
```
I would say - asking about displaying a static value which is defined by a user = meaningless: you may easily show any static text as a part of a title.

**Reply by ozMerv** [reply]
Im not asking to display a static value....

I want to display the value from aggregate_func : diff  --- which is dynamic as it increases as energy is consumed over teh day.

**Comment by ozMerv** [comment]
Im not asking to display a static value....

I want to display the value from aggregate_func : diff --- which is dynamic as it increases as energy is consumed over the day.

**Reply by ildar170975** [reply]
Can you test with
```
show:
  state: last
```
?
